### PR TITLE
#464 check and fix minor translation issues

### PR DESCRIPTION
**Pull request by sippsack**
Removed TODO and fixed some translations in headings and typos in text

### CODE REVIEW

File: docs/02-requirements/LZ-02-04.adoc
Comment (gernotstarke) [inline docs/02-requirements/LZ-02-04.adoc:1]: maybe "understand the Importance of Requirements for Software Architecture" or similar...

**Review by gernotstarke — CHANGES_REQUESTED** [review]
I suggest some changes, see detailed comments

**Comment by sippsack** [comment]
Thanks for your suggestions, will work them in shortly

File: docs/01-basics/LZ-01-04.adoc
[the remark this answers is not in the document]
Comment (sippsack) [reply docs/01-basics/LZ-01-04.adoc:1]: adjusted quite a few other LG titles

File: docs/01-basics/LZ-01-02.adoc
Comment (gernotstarke) [inline docs/01-basics/LZ-01-02.adoc:1]: sorry to be nitpicking. 

* Now we have title and content inconsistent:  In the title we use "Nutzen", in the text you use "Vorteile".
Please change both to  "Nutzen" 

* Grammar: we write "zu unterstützen", "zu erreichen"... but you removed the "zu" from "zu reduzieren". Please add that again

**Review by gernotstarke — CHANGES_REQUESTED** [review]
one minor issue concerning "Nutzen" and "Vorteile", plus one grammatical issue...

**Review by gernotstarke — APPROVED** [review]
thanx Falk!

**Pull request merged**